doozer/doozerlib/cli/images_konflux.py

```diff
@@ -34,6 +35,13 @@
 TRACER = trace.get_tracer(__name__)
 LOGGER = logging.getLogger(__name__)
 
+"""
+Including the following commands:
```

Comment:
This comment will get outdated very fast .. let's not introduce an overhead to maintain, that can confuse.  I would enthusiastically +1 a PR that splits these commands in their own files, but later ..